.mailmap

```diff
@@ -0,0 +1,13 @@
+Aleksandra Tarkowska <a.tarkowska@dundee.ac.uk> <A.Tarkowska@dundee.ac.uk> 
+Aleksandra Tarkowska <a.tarkowska@dundee.ac.uk> <aleksandra-tarkowska@users.noreply.github.com>
+Andreas Knab <andreas@glencoesoftware.com> <knabar@gmail.com>
+David Pinto <david.pinto@bioch.ox.ac.uk> <carandraug+dev@gmail.com>
+Helen Flynn <h.flynn@dundee.ac.uk> <omehelen@gmail.com>
```

Comment:
Same discussion as https://github.com/ome/openmicroscopy/pull/6349#discussion_r1240945862, there are multiple real names associated with this email address. If I want to switch to the most verbose

```
Proper Name <proper@email.xx> Commit Name <commit@email.xx>
```

form, this will need to be replaced by multiple entries